packages/ui-react/src/components/Player/Player.tsx

```diff
@@ -72,7 +72,16 @@ const Player: React.FC<Props> = ({
   const handleComplete = useEventCallback(onComplete);
   const handleUserActive = useEventCallback(onUserActive);
   const handleUserInactive = useEventCallback(onUserInActive);
-  const handleFirstFrame = useEventCallback(onFirstFrame);
+  const handleFirstFrame = useEventCallback(() => {
+    // a11y: auto focus to play/pause button
+    const playerContainer = playerRef.current?.getContainer();
+    const playPauseButton = playerContainer?.querySelector(
+      '.jw-controlbar [aria-label="Play"], .jw-button-container [aria-label="Pause"]',
+    ) as HTMLDivElement | null;
+    playPauseButton?.focus();
```

Comment:
We haven't run into this problem while testing. Can you elaborate?

Reply:
Try to open a video, hover on the back button and click on it.

Reply:
Hmm, I see what you mean. It's weird that we haven't noticed this before. 

It seems that there is a small race condition going on between the JW Player `userInactive` event when clicking the back button. I think we are only seeing this now because we have moved the focus to the controls and by clicking the back button, the player controls unfocus logic hides the UI immediately.

The `onMouseDown` event fires, but the `onClick` and `onMouseUp` events don't because the element is hidden.

I could solve it by delaying the `userInactive` event by 100ms, but relates directly to the time between the mouse down and mouse up events for this to work. Beceause the player fires the `userInactive` event on the onMouseDown event.

```tsx
  const handleUserInactive = useCallback(() => {
    setTimeout(() => setUserActive(false), 100);
  }, []);
```

Perhaps a better solution would be to prevent hiding the top bar when we detect a mouseDown on the back button.

Reply:
I fixed it properly by keeping the player overlay visible when it has focus. This also solves the issue that the overlay hides when navigating with the keyboard.

[7bb3f20](https://github.com/jwplayer/ott-web-app/pull/455/commits/7bb3f207ef794614d09b33d248dfcf4c7b220520)

Reply:
Works great! Thanks!